### DISCUSSION TITLE
Specify specific .NET SDK Version

### DISCUSSION THREAD

**Discussion by mumby0168** [dotnet/dotnet-docker#3553] [Q&A] [answered]
We are using this image to build and test a .NET 6 application snippet from this here

`FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base`

We have found an issue in version `6.0.201` we want to drop back to `6.0.200` how do I achieve this in the above image reference.

**Comment by mthalman** [comment, marked as answer]
Version 6.0.2 of ASP.NET Core was associated with version 6.0.200 of the SDK. So you can use `aspnet:6.0.2`.
